Felienne/hedy
# Code highlighting on for-teachers page was not level-dependent

The ace editor code examples on the for-teachers page where not highlighted. This was due to the structure where the keyword-highlighting was implemented based on `window.State.level`. An attribute that is not there on the for-teachers page. We solved this by adding the level as an id to the `<pre>` element in the .md-files and extract this value in the function that creates the ace editor.

**Description**

The ace editor on the for-teachers page now correctly highlights the keywords corresponding to the level for which the shown code is written. Better aligning the ace editor UI with the rest of the UI and make the code and mistakes shown better readable and understandable.

**Fix for**

Keyword not being highlighted on the for-teachers page when multi-line code examples where given. This PR fixed #727.

**How to test**

Navigate to for-teachers page and verify that the correct keywords are highlighted for the code examples and mistakes for different levels. Implemented for all languages except hu. This due to the structure not being correct and I'm unable to fix this due to a severe lack of understanding of the language.

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
